.github/workflows/ci-actions.yml

```diff
@@ -272,6 +272,41 @@ jobs:
           build-root-directory: integration-tests/gradle
           arguments: clean test -i -S --stacktrace --no-daemon
 
+  linux-jvm-maven-tests:
+    name: Maven Tests JDK ${{matrix.java.name}}
+    runs-on: ubuntu-latest
+    needs: build-jdk11
+    timeout-minutes: 60
+    strategy:
+      fail-fast: false
+      matrix:
+        java:
+          - {
+            name: Java 11,
+            java-version: 11
+          }
```

Comment:
If we only test JDK 11, couldn't we reuse the artifacts as we do with the native tests?

Reply:
Yeah, I just did a quick and dirty test here. I'll update it tomorrow

Reply:
Done